driver-core/src/main/java/com/datastax/driver/core/policies/WhiteListPolicy.java

```diff
@@ -61,4 +74,51 @@ public boolean apply(Host host) {
       }
     };
   }
+
+  /**
+   * Creates a new policy with the given host names.
+   *
+   * <p>See {@link #fromHosts(LoadBalancingPolicy, String...)} for more details.
+   */
+  public static WhiteListPolicy fromHosts(
+      LoadBalancingPolicy childPolicy, Collection<String> hostnames) {
```

Comment:
How about `Iterable<String>` instead of `Collection<String>`? (Then you could simply implement the logic here and make the overloaded method below call this one with `Arrays.asList(hostnames)`.)

Reply:
I used `Collection` for consistency with `Cluster.Builder.addContactPoints`, but I agree that using `Iterable` is better because it is less specific.